### PR TITLE
Fixed Typos.

### LINKED ISSUE #143: [FEATURE] Refactor code to remove typos
**Is your feature request related to a problem? Please describe.**
There are a few typos in naming variables and syntax mistakes which are ignored implicitly. One example will be `embededMessage` which must be spelled as `embeddedMessage`. I guess this was copy/pasted, so its spelled in this manner in many modules.

**Describe the solution you'd like**
Would like those and syntax mistakes to be fixed.

**Describe alternatives you've considered**
I guess I can't consider alternatives for this. But yea, I have a PR.

### PR DESCRIPTION
**Pull request by RazCrimson**
Closes #143

### CODE REVIEW

File: src/commands/multi-add.js [hunk 1]
```diff
@@ -20,16 +20,16 @@ module.exports = {
             })
 
             dbChannel.items.push(newItem)
-        })
+        }
```
Comment (rgroves) [inline src/commands/multi-add.js]: Why removing this parenthesis? Does the bot still work after this change (specifically the multi-add command)?

**Comment by RazCrimson** [comment]
I found some redundant initialization. And there was a loop that was made async unnecessarily in `/src/commands/multi-add.js`

Should I open another issue for that?

File: src/commands/poll.js [hunk 2]
```diff
@@ -44,16 +44,16 @@ module.exports = {
...
             pollDuration = Math.round(minutes * 60) * 1000 // rounding if minutes >0 and <1
 
-            // Creaete a nice, human-readable duration description (e.g. 2 hours and 30 minutes).
+            // Create a nice, human-readable duration description (e.g. 2 hours and 30 minutes).
             if (minutes > 59) {
                 let hours = Math.floor(minutes / 60)
-                let remainderMins = minutes % 60
+                let remainingMinutes = minutes % 60
                 let hourPluralizer = hours > 1 ? 's' : ''
-                let minPluralizer = remainderMins > 1 ? 's' : ''
```
Comment (rgroves) [inline src/commands/poll.js:52]: If you're going to change `remainderMins` to `remainingMinutes` might as well update `minPluralizer` to `minutePluralizer` as well to keep everything uniform.

**Review by rgroves — COMMENTED** [review]
Please also make sure you are testing the bot after all of your changes before submitting a PR. Right now there is no automated testing, so testing would just consist of running `$help` and then trying all of the commands on the list to ensure they all still work properly.

File: src/commands/multi-add.js (hunk 1 above)
Comment (RazCrimson) [reply src/commands/multi-add.js]: I tested the bot from my old branch, so I missed that. Sorry about that. Will be more careful next time.

File: src/commands/poll.js (hunk 2 above)
Comment (RazCrimson) [reply src/commands/poll.js:52]: Ok. Will do it.

**Comment by rgroves** [comment]
> I found some redundant initialization. And there was a loop that was made async unnecessarily in `/src/commands/multi-add.js`
> 
> Should I open another issue for that?

Open an issue and describe your findings (maybe link to the lines/files in question so we can talk about it) before moving forward. Thanks!

**Review by rgroves — COMMENTED** [review]
Okay. Looks good! Thanks.

**Comment by rgroves** [comment]
@tomassirio I reviewed the changes here and recommend accepting this PR.

**Pull request merged**